storage/watch.go

```diff
@@ -0,0 +1,54 @@
+package storage
+
+import (
+	"sync"
+
+	"github.com/coreos/etcd/storage/storagepb"
+)
+
+// Watcher uses to watch on key-value store.
+type Watcher interface {
```

Comment:
Yes. `Watcher(key []byte, prefix bool, startRev, endRev int64) (Watcher, CancelFunc)` CancelFunc is used to cancel watcher.